triggerReleasePipeline.sh

```diff
@@ -49,9 +49,6 @@ cloneGitHubRepo $ADOPTIUM_REPO
 # read expectedTag from cfg file (releasePlan.cfg) to see if this is the correct GA tag we want for release
 expectedTag=$(readExpectedGATag $JDKVERSION)
 
```

Comment:
maybe more this code a bit up, so we can bypass the check?
```suggestion
# fetch all new refs including tags from origin remote
cd "$WORKSPACE/$JDKVERSION"

# get latest -ga tag (sorted by time) in repo
latestGaTag=$(git tag --sort=-v:refname | grep '\-ga' | sort -V -r| head -1)
echo "Latest GA tag: ${latestGaTag}"

# from -ga tag find original commit SHA then list all tags point to the this SHA(exclude -ga tag) => orignal tag(s) from skara
# in rare case, there might be more tags than the one we want
# convert from multiple line string into Array and use the first one from the list which is supposed to have _adopt tag
scmReferenceString="$(git rev-list -1 ${latestGaTag} | xargs git tag --points-at  | grep  -v '\-ga')"
scmReferenceList=($scmReferenceString)
# append _adopt => release tag we use in adoptium
scmReference="${scmReferenceList[0]}_adopt"

# check if we need to proceed when scmReference has already triggered release pipeline in the past
checkPrevious ${scmReference}

# get the older tag name between the latest GA tag and expected tag
olderTag="$(echo -e "${expectedTag}\n${latestGaTag}" | sort -V | head -n1)"

# if GA tag (e.g jdk-19.0.2+5-ga) is greater or equal than expected (e.g jdk-19.0.2) => this is the correct GA we need
if [[ "${expectedTag}" == "${olderTag}" || "${expectedTag}" == "${latestGaTag}" ]]; then
  echo "We will proceed with ${latestGaTag} to trigger build"
else
  echo "${latestGaTag} is not the GA tag we expect for this release! We want e.g ${expectedTag}-ga or a higher GA tag"
  exit 0 # should not continue trigger logic
fi

```

Reply:
To bypass the check we should clear the "tracking" ?

Reply:
So say the previous trigger was "jdk-17.0.6+9_adopt" and for some reason we want it to trigger again, then we just clear the job's WORKSPACE ?

Reply:
bypass is just the move the checkPrevious() before` if [[ "${expectedTag}" == "${olderTag}" || "${expectedTag}" == "${latestGaTag}" ]]; then`

Reply:
ah I see what you've done now ! let me check

Reply:
if for any reason, we want to rebuild, we can wipe out workspace, or delete the `tracking` file

Reply:
yes agree